src/plays/simple-live-chat/components/chat/chat-body.scss

```diff
@@ -0,0 +1,107 @@
+@import "../../variables";
```

Comment:
the messages are getting scrolled behind the textbox. I would suggest keeping them in  separate containers and scroll only msg container

![image](https://user-images.githubusercontent.com/7145967/172046666-c4166934-b076-4543-88a3-e7c9f9b1a77f.png)

Reply:
yes this is now fixed.